client/css/style.css

```diff
@@ -893,6 +920,30 @@ kbd {
 	touch-action: pan-y;
 }
 
+#sidebar .join_form {
+	display: none;
+}
+
+#sidebar .join_form .input {
```

Comment:
I think most of this will become unnecessary if you remove `#window` from `#window .input` L713.

Reply:
done